feat(OHI IBM MQ): Added branching doc structure (#16140)
newrelic/docs-website: src/install/ibm-mq/linux/install-linux.mdx

---
componentType: default
headingText: Configure the IBM MQ monitoring integration
---

1. Change directory to the integrations configuration folder by running:

  ```shell
  cd /etc/newrelic-infra/integrations.d
  ```

2. Copy the sample configuration file by running:

  ```shell
  sudo cp ibmmq-config.yml.sample ibmmq-config.yml
  ```

> 3. Edit the `ibmmq-config.yml` file. This example config file collects all metrics and inventory from your localhost:

Reviewer: ```suggestion
3. Edit the `ibmmq-config.yml` file to collect all metrics and inventory from your local host:
```
